woocommerce/woocommerce
# Monorepo: git post-checkout hook.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Add git-hook to cacthup on node version and deps changes between branch switches

### How to test the changes in this Pull Request:

- execute `pnpm install --frozen-lockfile`
- test switching between this and trunk branches without preliminary `nvm use` - default pnpm is not available or not expected version, switching to this branch should prompt pnpm versions mismatches
